api/controllers/v0.2/createPatron.js

```diff
@@ -163,73 +261,52 @@ function createPatron(req, res) {
       .then((tokenResponse) => {
         if (!tokenResponse.data || !tokenResponse.data.access_token) {
           reject();
+          return;
+        }
+        callAxiosToCreatePatron(req, res, tokenResponse);
+      })
+      .catch((tokenErrorResponse) => {
+        console.error(
+          `status_code: ${tokenErrorResponse.response.status}, ` +
+          'type: "invalid-request", ' +
+          `message: "${tokenErrorResponse.message} from ILS.", ` +
+          `response: ${JSON.stringify(tokenErrorResponse.response.data)}` // eslint-disable-line comma-dangle
+        );
+
+        if (tokenErrorResponse.response && tokenErrorResponse.response.data) {
+          const responseObject = collectErrorResponseData(
+            tokenErrorResponse.response.status,
+            'invalid-request',
+            `${tokenErrorResponse.message} from ILS.`,
+            'ILS Response',
+            tokenErrorResponse.response.data // eslint-disable-line comma-dangle
+          );
+
+          const statusCode = (responseObject.status) ? responseObject.status : 500;
```

Comment:
Unnecessary ternary. could be `const statusCode = responseObject.status || 500;`